go.mod

```diff
@@ -59,7 +59,7 @@ require (
 )
 
 require (
-	github.com/0xPolygon/go-ibft v0.4.0
+	github.com/0xPolygon/go-ibft v0.4.1-0.20230418151118-337b82c3c3c4
```

Comment:
Should we make a new release 0.4.1 on go-ibft and refer it that way?

Reply:
That can be done after the release of go-ibft